src/game/word_match/systems.rs

```diff
@@ -47,7 +48,13 @@ pub fn check_matches(
         }
 
         if text.sections[1].value.is_empty() {
-            event_writer.send(WordEvent(word.0));
+            #[allow(clippy::single_match)]
```

Comment:
```
if let WordTarget::Enemy(id) = word.0 {
     enemy_event_writer.send(EnemyShotEvent(id));
}
```

Reply:
Leave it as is if you plan to add more matches later on

Reply:
Ya eventually this will match for different events such as Heal, reload, move